Cargo.toml

```diff
@@ -50,8 +46,8 @@ tokio = { version = "1.35.1", features = [
     "time",            # anables `tokio::time` namespace holding the `sleep` utility and `Duraction` struct
 ] }
 tokio-native-tls = { version = "0.3.1", optional = true }
-tokio-rustls = { version = "0.25.0", optional = true }
-rustls-native-certs = { version = "0.7.1", optional = true }
+tokio-rustls = { version = "0.26", optional = true }
+rustls-platform-verifier = { version = "0.4", optional = true }
```

Comment:
In their `0.8`, `rustls-native-certs` introduced changes that are [breaking](https://github.com/rustls/rustls-native-certs/commit/8aa350a8a0f02accd5be4652975a3fbc1b69bf97#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759L120-R135) for us. 

We are now also taking into account the call from their [README](https://github.com/rustls/rustls-native-certs?tab=readme-ov-file)
"""
Instead of this crate, we suggest using [rustls-platform-verifier](https://github.com/rustls/rustls-platform-verifier), which provides a more robust solution with a simpler API. This crate is still maintained, but mostly for use inside the platform verifier on platforms where no other solution is available. 
"""